eml/components/explore/ExploreCard.jsx

```diff
@@ -1,34 +1,224 @@
-import { View, Text, Image, Pressable } from 'react-native'
-import React from 'react'
-import { useFonts, VarelaRound_400Regular } from '@expo-google-fonts/dev'
-import { useNavigation } from '@react-navigation/native'
-import { AppLoading } from 'expo-app-loading'
+import React, { useEffect } from "react";
+import { View, Text, Image, Pressable } from "react-native";
```

Comment:
Update to use the new Text component

Reply:
@Educado-App/video-streaming-team can you look at this since you have made the ExploreCard